src/tests/data/png.dfdl.xsd

```diff
@@ -0,0 +1,764 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As I have learned from @stevedlawrence , do not add this file into the repo. Its license is not compatible with ALv2 I believe. You will instead want to make a custom schema for testing, something similar to what we have in `main`.